source/changelog.txt

```diff
@@ -12,6 +12,71 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.2.1
+------
+
+*Released March 19, 2024*
+
+Fixes a bug in 2.2.0 where ``require('<module>')`` caused an error in 
+script mode:
+
+- :issue:`MONGOSH-1738` - ``require`` not working in script mode.
+
+v2.2.0
+------
+
+*Released March 11, 2024*
+
+.. important::
```

Comment:
[Suggestion]

Given that we're highlighting a bug, the `warning` direcive may be more appropriate.